Formula/clazy.rb

```diff
@@ -0,0 +1,69 @@
+class Clazy < Formula
+  desc "Qt oriented static code analyzer"
+  homepage "https://www.kdab.com/"
+  url "https://download.kde.org/stable/clazy/1.9/src/clazy-1.9.tar.xz"
+  sha256 "4c6c2e473e6aa011cc5fab120ebcffec3fc11a9cc677e21ad8c3ea676eb076f8"
+  license "LGPL-2.0-or-later"
+  head "https://invent.kde.org/sdk/clazy.git"
+
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake"   => [:build, :test]
+  depends_on "llvm@11" => [:build, :test]
```

Comment:
When will this be compatible with LLVM 12?

Reply:
1.10, if use `llvm`, the audit step will be failed, macOS llvm provides only `clang` executable, `libclang`, `llvm-binutils`, `libc++`.

Reply:
Are you sure this can't use the `llvm` formula? The KDE tap uses `llvm` and not `llvm@11`. https://invent.kde.org/packaging/homebrew-kde/-/blob/master/Formula/clazy.rb

Reply:
Adding upstream patch to support `llvm`.